.github/workflows/github-actions-demo.yml

```
@@ -4,26 +4,24 @@ jobs:
   Explore-GitHub-Actions:
     runs-on: ubuntu-latest
     env:
-      GILAB_TOKEN: Lr6mLevgRbmEpEhmzojo
+      GILAB_TOKEN: ${{ secrets.GILAB_TOKEN }}
       PATH_DESTINATION: dist
+      USER_GITLAB: ${{ secrets.USER_GITLAB }}
+      EMAIL_GITLAB: ${{ secrets.EMAIL_GITLAB }}
 
     steps:
       - uses: actions/checkout@v1
       - run: git clone https://oauth2:$GILAB_TOKEN@gitlab.luizalabs.com/kaique.feng/test-pipeline.git open-api-gitlab
       - run: ls
-      - name: Create new branch and commit changes
+      - name: Create new release content
         working-directory: open-api-gitlab
         run: |
-          git checkout -b devportal-ci-github
-          ls
-          test -e $PATH_DESTINATION && echo File Extist || mkdir $PATH_DESTINATION
-          cp ../api-reference/*.yml $PATH_DESTINATION/production/
-          git --no-pager diff
-          git config --global user.name kaique.feng
-          git config --global user.email kaique.feng@luizalabs.com
-          git add .
-          git commit -m "included new open api"
-      - name: Push and open merge request
-        working-directory: open-api-gitlab
-        run: |
-          git push origin devportal-ci-github --force -o merge_request.create -o merge_request.label="New PR CI/CD"
+          git fetch --all --tags
+          tag=$(git describe --tags `git rev-list --tags --max-count=1`)
+          tagName=($(echo "$tag" | sed "s|\(.*\)[.].*|\1.|"))
+          version=($(echo "$tag" | grep -oE "[^.]*$"))
```

```suggestion
          version=($(echo "$tag" | grep -oE "[^.]+$"))
```
tem que ter no mínimo uma ocorrência!

---

Ditto: Adicionaria um comentariozinho com exemplo pra deixar claro o objetivo.

Regex are hard. ;-)